package.json

```diff
@@ -8,15 +8,18 @@
     "lint": "eslint '**/*js'",
     "lint:fix": "prettier-eslint --write '**/*js' ",
     "start-dev": "nodemon ./server/app.js --exec babel-node",
+    "start": "babel-node ./server/app.js --exec",
```

Comment:
Do not use `babel-node` to build production servers. Go and do some research on this